app/screens/invite/summary_report.tsx

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {useIntl} from 'react-intl';
+import {View, Text} from 'react-native';
+
+import CompassIcon from '@components/compass_icon';
+import UserItem from '@components/user_item';
+import {useTheme} from '@context/theme';
+import {makeStyleSheetFromTheme, changeOpacity} from '@utils/theme';
+import {typography} from '@utils/typography';
+
+import {SearchResult, InviteResult} from './invite';
+import TextItem, {TextItemType} from './text_item';
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
+    return {
+        summaryInvitationsContainer: {
```

Comment:
all of these contain a `summaryInvitations` prefix and I would suggest removing it from each of the field names.

Reply:
Removed